packages/lab/src/button/Button.tsx

```diff
@@ -309,6 +309,12 @@ const ButtonRoot = styled.button<ButtonStyleProps>(
               --button-text-color: ${variables.textColorHovered};
               --button-border-color: ${variables.borderColorHovered};
               --button-background-color: ${variables.backgroundColorHovered};
+
+              span {
+                svg {
```

Comment:
Lets make this selector more specific. It may override any svg icon

Reply:
@doniyor2109 but we will use only hover moment and we have only startIcon and endIcon. We can not make any specific here, because of we have custom icons and mui svg icons.

Reply:
So we should make apply it for all icons?

Reply:
yes

Reply:
@doniyor2109 it affects only on hover moment. And I have checked button only does have startIcon and endIcon. That's why there won't be any issue with other moments. Only icon color will be the same as text on hover moment.